ClientTests/URIFixupTests.swift

```diff
@@ -73,11 +73,14 @@ class URIFixupTests: XCTestCase {
         }
     }
     
-    func testValidURLsForNoSchemes() {
+    func testValidURLsForNoSchemes() throws {
+        XCTSkip("No valid URL")
```

Comment:
@isabelrios `URIFixup` may behave differently on XCode 15. 😞 Looks like I can't update these tests without the developers.

Reply:
This is a unit tests, you would need their help.

Reply:
I can take a look at it right now

Reply:
Thank you @PARAIPAN9 !

Reply:
@clarmso looks like @PARAIPAN9 can't reproduce this test failure locally, could you please enable it and run bitrise to see what's the failure? Thanks!

Reply:
Hm ok. The tests pass on Bitrise but fail on my end. @isabelrios Could you please approve this PR? I can't merge the PR unless one code owner approves.

https://app.bitrise.io/app/f9ba050939a50ea4/pipelines/739f3509-dbbd-477e-af72-22a228cfb964

Reply:
If this is ready, just by adding the label, the merge will happen automatically